grafana/grafana.libsonnet

```diff
@@ -295,10 +295,7 @@
         readinessProbe: {
           httpGet: { path: '/api/health', port: portName },
         },
-        resources: {
-          requests: { cpu: '100m', memory: '100Mi' },
-          limits: { cpu: '200m', memory: '200Mi' },
-        },
+        resources: $._config.grafana.container,
```

Comment:
actually .. could you rename this field from `container` to `resources`? I'm not sure how that got in the configuration this way

Reply:
Yep sure, I was thinking the same thing but was worried others code might already use this part. Will do in a sec